sdk/servicebus/service-bus/test/streamingReceiver.spec.ts

```diff
@@ -700,7 +697,7 @@ describe("Streaming - Settle an already Settled message throws error", () => {
     await afterEachTest();
   });
 
-  const testError = (err: Error, operation: DispositionType) => {
+  const testError = (err: Error, operation: DispositionType): void => {
```

Comment:
What should be appropriate return type for `testError` ? @ramya-rao-a

Reply:
void is right